README.md

```diff
@@ -9,20 +9,25 @@ many pieces. It can drive a very wide range of build and packaging systems,
 so that you can simply list all the upstream projects you want and have
 them built and installed together as a single tree.
 
+![Snapcraft Overview][overview-image]
```

Comment:
Is this diagram your creation? Nice if so.

I'd replace the *package yaml* with *package metadata*